src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -0,0 +1,711 @@
+                            Navigation.navigate(
+                                ROUTES.MONEY_REQUEST_STEP_TAG.getRoute(action, iouType, index, transactionID, reportID, Navigation.getActiveRouteWithoutParams(), reportActionID),
+                            )
+                        }
+                        style={[styles.moneyRequestMenuItem]}
+                        disabled={didConfirm}
+                        interactive={!isReadOnly}
+                        rightLabel={isTagRequired && canUseViolations ? translate('common.required') : ''}
+                    />
+                ),
+                shouldShow: shouldShowTags,
+                isSupplementary: !isTagRequired,
+            };
+        }),
+        {
+            item: (
+                <MenuItemWithTopDescription
+                    key={`${taxRates?.name}${taxRateTitle}`}
+                    shouldShowRightIcon={canModifyTaxFields}
+                    title={taxRateTitle}
+                    description={taxRates?.name}
+                    style={[styles.moneyRequestMenuItem]}
+                    titleStyle={styles.flex1}
+                    onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_TAX_RATE.getRoute(action, iouType, transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    disabled={didConfirm}
+                    interactive={canModifyTaxFields}
+                />
+            ),
+            shouldShow: shouldShowTax,
+            isSupplementary: true,
+        },
+        {
+            item: (
+                <MenuItemWithTopDescription
+                    key={`${taxRates?.name}${formattedTaxAmount}`}
+                    shouldShowRightIcon={canModifyTaxFields}
+                    title={formattedTaxAmount}
+                    description={translate('iou.taxAmount')}
+                    style={[styles.moneyRequestMenuItem]}
+                    titleStyle={styles.flex1}
+                    onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_TAX_AMOUNT.getRoute(action, iouType, transactionID, reportID, Navigation.getActiveRouteWithoutParams()))}
+                    disabled={didConfirm}
+                    interactive={canModifyTaxFields}
+                />
+            ),
+            shouldShow: shouldShowTax,
+            isSupplementary: true,
+        },
+        {
+            item: (
+                <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, styles.ml5, styles.mr8, styles.optionRow]}>
+                    <ToggleSettingOptionRow
+                        switchAccessibilityLabel={translate('common.billable')}
+                        title={translate('common.billable')}
+                        onToggle={(isOn) => onToggleBillable?.(isOn)}
+                        isActive={iouIsBillable}
+                        disabled={isReadOnly}
+                        titleStyle={!iouIsBillable && {color: theme.textSupporting}}
```

Comment:
The conditional titleStyle here led to this issue:
- https://github.com/Expensify/App/issues/53460

which was fixed by removing the style completely.